integration-tests/src/test/java/org/openmrs/maven/plugins/SetupIntegrationTest.java

```diff
@@ -215,6 +215,26 @@ public void setup_shouldInstallServerWithGivenJavaHomeAndAddJavaHomeToSdkPropert
         assertThat(server, serverHasDebugPort("1044"));
     }
 
+    @Test
+    public void setup_shouldInstallWithParentDistroSpecifiedInDistroProperties() throws Exception{
```

Comment:
@rkorytkowski We tested this manually by making fake releases locally. Its difficult to do integration tests for this since we don't have any released distro with specified parent in `openmrs-distro.properties`